go.mod

```diff
@@ -1,35 +1,38 @@
 module github.com/bitcoin-sv/go-paymail
 
-go 1.18
+go 1.21.3
```

Comment:
Shouldn't we use `1.21.3`  (or `1.21.5`) in other libs too?

Reply:
Good question. I posted a little discussion regarding more-or-less that topic on our channel.

The main thing is, the version we specify here is the version that people who use our libraries will be forced to install. So IMO, we should upgrade the version to the newest "major" - 1.21, but not necessarily to the newest patch (1.21.5), because we don't care if the user uses 1.21.1 or 1.21.5.

In this repo it is necessary to have this version, because one of the dependencies requires it and the update happened automatically when I did `go get -u ./...`.

Does this make any sense?

Reply:
Updated to 1.21.5 :)